src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    const allDirectors = movies.map(item => item.director)
+    return allDirectors
+}
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+const howManyMovies = (movies) => {
+    let spielbergFilms = movies.filter(
+      films => films.director.includes("Steven Spielberg"))
+
+    let dramaFilms = spielbergFilms.filter(
+      films => films.genre.includes("Drama"))
+    return dramaFilms.length
+  }
+  
+  howManyMovies(movies)
+
 // Iteration 3: All rates average - Get the average of all rates with 2 decimals
 
+// Q: how to access object inside array?
+//NO NEED! use map to get all rates on new array!
+
```

Comment:
You should put line 27 inside function ratesAverage, as it's meant to be used only in this scope.